gradle.properties

```diff
@@ -37,4 +37,4 @@ POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=paystack
 POM_DEVELOPER_NAME=Paystack
 POM_DEVELOPER_EMAIL=developers@paystack.co
-org.gradle.unsafe.configuration-cache=true
\ No newline at end of file
+org.gradle.unsafe.configuration-cache=false
```

Comment:
Were you having problems with this optimisation?

Reply:
Some plugins used aren't compatible. I don't want to have to update them at this time.